README.md

```diff
@@ -27,8 +27,15 @@ Instructions for creating and deploying a new App Configuration Policy can be fo
 1. Enter the Bundle ID of your app and click **OK**. The Chatr bundle ID can be found by selecting the project file in the Xcode project explorer, selecting the chatr target, and selecting the "General" tab. This app's Package ID is `Intune.chatr`.
 1. Click the **Configuration settings** button in the "Add configuration policy" pane and set the key-value pair configuration you would like to apply to a user group for your app. For intance, to change the messaging group name on the Chat Page of the Chatr sample app to "Intune", you can create a configuration where the key is "GroupName" and the value is "Intune".
 1. Once you have added the key-value pair configuration click **OK** at the bottom of the "Configuration" pane and then click **Add** at the bottom of the "Add configuration policy" pane. Your app should now appear in the "App configuration policies" pane.
-    
-### Step 4: Launch the App & Sign-In
+
+### Step 4: Register the App with Microsoft Entra ID
+1. You'll need to configure an app registration in Microsoft Entra ID and specify the client ID and redirect URI that the Intune SDK should use. Instructions for creating an app registration can be found [here](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app).
+2. This app's bundle ID is `Intune.chatr`. Make sure you update this bundle ID to match the app registration. The bundle ID can be found by selecting the project file in the Xcode project explorer, selecting the chatr target, and selecting the "General" tab.
+3. In the Info.plist file, update the values for `ADALClientId` and `ADALRedirectUri` under IntuneMAMSettings dictionary to match the client ID and redirect URI from your app registration. The redirect URI should be in the format `msauth.<bundle-id>://auth`. 
+4. If your app registration was configured as a single-tenant app, you will also need to add the `ADALAuthority` value to the Info.plist file under IntuneMAMSettings. This can be skipped for multi-tenant app registrations.
+5. The redirect URI scheme which is of the format `msauth.<bundle-id>` needs to be added to the `URL Schemes` under `URL Types` in the Info.plist file.
```

Comment:
@vigavadi - Could you include a step here for granting the app access to the Intune service in any tenants the user intends to test with? Could you also call out that the redirect URI scheme will need to be added to the CFBundleURLSchemes list in the Info.plist if the app's bundle ID is changed?

Reply:
Link to relevant docs for granting access to the Intune service: https://learn.microsoft.com/intune/intune-service/developer/app-sdk-get-started#give-your-app-access-to-the-intune-mobile-app-management-service